module/src/test/scala/org/scalatestplus/play/OneAppPerSuiteComponentSpec.scala

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2001-2016 Artima, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.scalatestplus.play
+
+import org.scalatest.{ ConfigMap, Outcome }
+import org.scalatestplus.play.components.OneAppPerSuiteWithComponents
+import play.api.mvc.Result
+import play.api.test.Helpers._
+import play.api.test.{ FakeRequest, Helpers }
+import play.api.{ Application, BuiltInComponents, BuiltInComponentsFromContext, Play }
+
+import scala.concurrent.Future
+
+class OneAppPerSuiteComponentSpec extends UnitSpec with OneAppPerSuiteWithComponents {
+
+  override def components: BuiltInComponents = new BuiltInComponentsFromContext(context) {
+
+    import play.api.mvc.{ Action, Results }
+    import play.api.routing.Router
+    import play.api.routing.sird._
+
+    lazy val router: Router = Router.from({
+      case GET(p"/") => Action {
+        Results.Ok("success!")
+      }
+    })
+
+    override lazy val httpFilters = Seq()
```

Comment:
You can use `new BuiltInComponentsFromContext(context) with play.api.NoHttpFiltersComponents` instead. This is how it is documented in Play, so it would be good to do the same here.